### ISSUE TITLE
TypeScript compilation error: "Generic type 'Array<T>' requires 1 type argument(s)"

### ISSUE THREAD

**Issue by itsdevdom** [octokit/octokit.js#576]
Since version 9.3, the TypeScript compiler (in my case the latest 2.4.x) throws two errors regarding a missing generic for Arrays. In particular:

**First error:**
```
node_modules/github/lib/index.d.ts(1463,24): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
```
Here the line:
https://github.com/mikedeboer/node-github/blob/f34f59e2ea8d0c03c0a449e1c9f65de2fe2bb413/lib/index.d.ts#L1463

**Second error:**
```
node_modules/github/lib/index.d.ts(1471,24): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
```
Here the line:
https://github.com/mikedeboer/node-github/blob/f34f59e2ea8d0c03c0a449e1c9f65de2fe2bb413/lib/index.d.ts#L1471

**Solution:**

I assume using ```string[]``` or ```Array<string>``` in both cases should fix the issue.

**Comment by itsdevdom** [comment]
Opened a PR with the solution described above; also tested it within my project and it seems to fix the TypeScript compilation issue.

**Comment by kaizensoze** [comment]
I think this is a result of adding a `team_reviewers` param in routes.json and then running `node lib/generate.js`. I'll look into changing the default from `Array` to `string[]`. There's another block of code in the file that has `Array` that I'll look into:

```
export type ReposReplaceTopicsParams =
    & Owner
    & Repo
    & {
      names: Array;
    };
```

**Comment by itsdevdom** [comment]
I've updated my PR, fixing the issue in the `generate.js` file instead (as @kaizensoze mentioned). Also, it now uses the correct type for `ReposReplaceTopicsParams`.

Does https://github.com/mikedeboer/node-github/pull/577 look alright?

**Comment by elldritch** [comment]
Can this fix be published as 9.3.2? I need the TypeScript support but am not in a position to upgrade major versions.
